src/resources/views/base/inc/head.blade.php

```diff
@@ -9,21 +9,15 @@
     @yield('before_styles')
     @stack('before_styles')
 
-    @if (config('backpack.base.styles') && count(config('backpack.base.styles')))
-        @foreach (config('backpack.base.styles') as $path)
+    @foreach(config('backpack.base.styles', []) as $path)
         <link rel="stylesheet" type="text/css" href="{{ asset($path).'?v='.config('backpack.base.cachebusting_string') }}">
-        @endforeach
-    @endif
+    @endforeach
 
-    @if (config('backpack.base.mix_styles') && count(config('backpack.base.mix_styles')))
-        @foreach (config('backpack.base.mix_styles') as $path => $manifest)
+    @foreach(config('backpack.base.mix_styles', []) as $path => $manifest)
         <link rel="stylesheet" type="text/css" href="{{ mix($path, $manifest) }}">
-        @endforeach
-    @endif
+    @endforeach
 
```

Comment:
The changes above are not required, but this got so much cleaner!
If we default the config to an empty array, we no longer need to 1. check if key exists, 2. check if it has count.